### ISSUE TITLE
Swin-UNETR pretraining validation criterion

### ISSUE THREAD

**Issue by Tkorsten** [Project-MONAI/research-contributions#370]
Hi all,

I was working on pretraining the Swin-UNETR model using a private dataset, however, I was wondering if there was a particular reason for the fact that only the val_loss_recon (the inpainting loss for the validation set) is used as a criterion to decide whether or not to store the model during pretraining. If all three pretraining losses are accounted for during training, then why only focus on one during the saving of the best model?

Any clarification upon this choice will be greatly appreciated.
